kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml

```diff
@@ -165,6 +170,16 @@ domainHomeImageBase: container-registry.oracle.com/middleware/weblogic:12.2.1.4
 # Default is /tmp/dhii-sample/tools
 # toolsDir: /tmp/dhii-sample/tools
 
+# WLST script that the create domain script uses to create a WebLogic domain
+# when using wlst mode.
+# This value is ignored when WDT is used in creating a WebLogic domain.
+createDomainWlstScript: wlst/create-wls-domain.py
+
+# WDT model YAML file that the create domain script uses to create a 
+# WebLogic domain when using wdt mode.
+# This value is ignored when WLST script is used in creating a WebLogic domain.
+#createDomainWdtModel: wdt/wdt_model_dynamic.yaml
```

Comment:
I find it a little odd that the WlstScript is uncommented and WdtModel is commented; instead, can you follow the same approach for both settings?  

Also, I assume WdtModel defaults to wdt/wdt_model_dynamic.yaml? If so, please add a comment here? 

Does WlstScript have a default?  It should IMO so as to match the WdtModel pattern, and it should be similarly documented.

Reply:
Thanks. both are uncommented now and added comments on what the defaults values are if not specified.